open-build-service/notqmail.spec.in

```diff
@@ -0,0 +1,365 @@
+#log directory if we use multilog in supervise scripts
+%global logdir             /var/log/svc
+%global see_base           For a description of notqmail visit https://notqmail.org/
+
+Name: notqmail
+Version: @version@ 
+Release: @release@%{?dist}
+Summary: A community driven fork of qmail
+License: CC-PDDC
+URL: https://notqmail.org
+Source0: https://github.com/notqmail/notqmail/releases/download/notqmail-@version@/notqmail-@version@.tar.xz
+%if 0%{?suse_version} >= 1120
+Source1: %{name}-permissions.easy
+Source2: %{name}-permissions.secure
+Source3: %{name}-permissions.paranoid
+%endif
+Source4: system-users-qmail.conf
+%if %{undefined suse_version} && %{undefined sles_version}
+Group: System Environment/Base
+%else
+Group: Productivity/Networking/Email/Servers
+%endif
+BuildRequires: rpm gcc make coreutils
+BuildRequires: glibc glibc-devel
+
+%if 0%{?fedora_version} || 0%{?centos_version} || 0%{?rhel_version}
+Requires(pre): shadow-utils
+Requires(postun): shadow-utils
+%endif
+%if 0%{?suse_version} || 0%{?sles_version}
+Requires(pre): pwdutils
+Requires(postun): pwdutils
+%endif
+%if %{undefined fedora_version} && %{undefined centos_version} && %{undefined rhel_version} && %{undefined sles_version} && %{undefined suse_version} && %{undefined scientificlinux_version} && %{undefined openeuler} && %{undefined almalinux_version} && %{undefined rocky_version}
+BuildRequires: sysuser-tools
+%endif
+%if 0%{?suse_version} >= 1500 || 0%{?sles_version} >= 15
+BuildRequires: sysuser-tools
+%endif
+
+##################################### OBS ####################################
+%if %build_on_obs == 1
+%if 0%{?suse_version}
+BuildRequires: -post-build-checks  
+#!BuildIgnore: post-build-checks  
+%endif
+%endif
+##############################################################################
+
+Requires: /usr/sbin/useradd /usr/sbin/userdel /usr/sbin/groupadd /usr/sbin/groupdel
+Requires: /sbin/chkconfig procps
+Requires: coreutils /bin/sh glibc
+%if "%{?_unitdir}" == ""
+Requires: initscripts
+%endif
+%if 0%{?suse_version} >= 1120
+PreReq: permissions
+%endif
+
+Provides: user(alias)       > 999
+Provides: user(qmaild)      > 999
+Provides: user(qmaill)      > 999
+Provides: user(qmailp)      > 999
+Provides: user(qmailq)      > 999
+Provides: user(qmailr)      > 999
+Provides: user(qmails)      > 999
+Provides: group(nofiles)    > 999
+Provides: group(qmail)      > 999
+Provides: smtp_daemon
+%if %{undefined fedora_version} && %{undefined centos_version} && %{undefined rhel_version} && %{undefined sles_version} && %{undefined suse_version} && %{undefined scientificlinux_version} && %{undefined openeuler} && %{undefined almalinux_version} && %{undefined rocky_version}
+%sysusers_requires
+%endif
+%if 0%{?suse_version} >= 1500 || 0%{?sles_version} >= 15
+%sysusers_requires
+%endif
+
+%if %build_on_obs == 1
+BuildRoot: %{_tmppath}/%{name}-%{version}-build
+%endif
+
+%description
+notqmail is a community-driven fork of qmail, beginning where netqmail
```

Comment:
This description is in a few places. Should be updated to the latest from the notqmail.org front page:

> notqmail is the collaborative Open Source successor to [qmail](https://cr.yp.to/qmail.html) and [netqmail](http://netqmail.org/). It begins with stable, compatible, small [releases](https://notqmail.local/releases/) to which existing qmail users can safely [update](https://notqmail.local/install/), and [aims to become](https://notqmail.local/roadmap/) a [more extensible](https://notqmail.local/designs/), [more easily packaged](https://notqmail.local/goals-and-non-goals/), and [more modern](https://notqmail.local/feature-wishlist/) Mail Transport Agent suitable for most needs.

Reply:
@DerDakon one can use a macro to define a string and then use that macro below %description. Will have to try this out (in the next version) without causing any syntax error

Reply:
@schmonz , ATM I'm just replacing the description with the updated one as above. For the next version I will look at using sed, head, tail etc to pick the description from README or a common file

Reply:
This has been addressed in the latest commit #[3eaf213](https://github.com/notqmail/notqmail/pull/76/commits/3eaf2138ce40da0831bfc90852faa8b7c2a2c4ff)